requirements.txt

```diff
@@ -1,3 +1,3 @@
 -e .
 # DON'T FORGET TO UPDATE setup.py !!
```

Comment:
I think we forgot to update `setup.py` 😹

Reply:
😅 woops 🤦

Reply:
On the other hand, the library still works with version 52.2.0 of dmutils, it's just the tests that change, so maybe the version in setup.py could stay the same? What do you think?

Reply:
I'd rather keep them consistent if we can.